fish-rust/src/ffi.rs

```diff
@@ -136,6 +140,9 @@ impl From<wcharz_t> for &wchar::wstr {
 /// Allow wcharz_t to be "into" WString.
 impl From<wcharz_t> for wchar::WString {
     fn from(w: wcharz_t) -> Self {
+        if w.str_.is_null() {
```

Comment:
We shouldn't silently convert null strings to empty; I've added an assertion on the C++ side here.

Probably the simplest thing to do here is just pass an empty string on the C++ side, instead of null.

Reply:
I'm now using an empty string.